### PR TITLE
test: remove deprecations for symfony > 5.1 in AppKernel

### PR DESCRIPTION
**Pull request by art-cg**
There are different possible solutions to handle the deprecations.
I do not know which one you prefer.

To reproduce remove remove this line from phpunit.xml
`<env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />`

The main goal is to prepare for symfony 6

### CODE REVIEW

File: tests/Fixtures/App/AppKernel.php [hunk 1]
```diff
@@ -11,33 +11,64 @@
...
+                new FrameworkBundle(),
+                new TwigBundle(),
+                new ApiPlatformBundle(),
+                new ApiPlatformMsBundle(),
+            ];
+        }
+
+        protected function configureRoutes(RouteCollectionBuilder $routes): void
+        {
+            $routes->import(__DIR__.'/config/routing.yml');
```
Comment (mtarld) [inline tests/Fixtures/App/AppKernel.php:36]: I may have a simpler solution. As `RoutingConfigurator` and `RouteCollectionBuilder` provide to us the same method `import` (in our scope), we can do the following:
```php
protected function configureRoutes(/* RouteCollectionBuilder|RoutingConfigurator $routes */): void
{
    /** @var RouteCollectionBuilder|RoutingConfigurator $routes */
    $routes = func_get_arg(0);

    $routes->import(__DIR__.'/config/routing.yml');
}
```

WDYT?

**Review by mtarld — CHANGES_REQUESTED** [review]

File: tests/Fixtures/App/AppKernel.php (hunk 1 above)
Comment (art-cg) [reply tests/Fixtures/App/AppKernel.php:36]: Sounds good. I will test it

Comment (art-cg) [reply tests/Fixtures/App/AppKernel.php]: I just changed it to `/* RouteCollectionBuilder|RoutingConfigurator */ $routes` and it works.

Comment (mtarld) [reply tests/Fixtures/App/AppKernel.php]: Therefore you can do:
```php
    /**
     * @param RouteCollectionBuilder|RoutingConfigurator $routes
     */
    protected function configureRoutes($routes): void
    {
        $routes->import(__DIR__.'/config/routing.yml');
    }
```

Comment (art-cg) [reply tests/Fixtures/App/AppKernel.php]: Ok, i updated it like this now

**Review by mtarld — APPROVED** [review]

**Comment by mtarld** [comment]
Thank you @art-cg!

**Pull request merged**